src/main/kotlin/no/nav/familie/tilbake/integration/familie/IntegrasjonerClient.kt

```diff
@@ -60,4 +69,120 @@ class IntegrasjonerClient(@Qualifier("azure") restOperations: RestOperations,
     fun hentOrganisasjon(organisasjonsnummer: String): Organisasjon {
         return getForEntity<Ressurs<Organisasjon>>(hentOrganisasjonUri(organisasjonsnummer)).getDataOrThrow()
     }
+
+    fun opprettOppgave(opprettOppgave: OpprettOppgaveRequest): String {
+        val uri = URI.create(integrasjonerConfig.integrasjonUri.toString() + "/oppgave/opprett")
```

Comment:
Alle de ander pathsegmentene mot integrasjoner ligger i integrasjonerConfig som konstanter. Synes vi burde samle dem på ett sted i stedet for å ha dem som spredte fritekster. Kanskje også defienere uri sammen med de andre uriene øverst i filent?

Reply:
👍